mock/server/worker.go

```diff
@@ -31,6 +31,23 @@ const (
 			"last_checked_in": 1602612590
 		}`
 
+	// UpdateWorkerResp represents a JSON return for a single updated worker.
+	UpdateWorkerResp = `
+		{
+			"worker": {
+				"id": 1,
+				"hostname": "worker_1",
+				"address": "http://vela:8080",
+				"routes": [
+				"large",
+				"docker",
+				"large:docker"
+				],
+				"active": true,
+				"last_checked_in": 1602612590
+			}
+		}`
```

Comment:
```suggestion
	UpdateWorkerResp = `
		{
			"id": 1,
			"hostname": "worker_1",
			"address": "http://vela:8080",
			"routes": [
				"large",
				"docker",
				"large:docker"
			],
			"active": true,
			"last_checked_in": 1602612590
		}`
```

Reply:
needs the wrapper field removed.
i might have messed up the whitespace in there via, it needs to be double checked in an editor